### ISSUE TITLE
Issue with linking extract in Personal Project Portfolio

### ISSUE THREAD

**Issue by zenghou** [nus-cs2103-AY1718S1/forum#202]
Hello everyone, 

I am working on the PPP and I am trying to link the extracts from both UserGuide and DeveloperGuide. In both documents, I have tagged the respective sections with `// tag::exampleFeature[]` and the closing tag `// end::exampleFeature[]`. However, when I try to link the extract in the PPP using `include::../DeveloperGuide.adoc[tag=exampleFeature]`, I get a link to the guide instead of a excerpt of the guide. 

<img width="367" alt="screen shot 2017-11-09 at 11 39 41 pm" src="https://user-images.githubusercontent.com/23148645/32614021-491068ba-c5a7-11e7-8149-caa0eb0848ab.png">

Is anyone else experiencing this issue as well, or did I tag the sections wrongly? Thanks in advance!

**Comment by karrui** [comment]
It's normal, converting it to a PDF allows everything to be properly displayed (But don't commit it to GitHub, Travis will complain)

Regards, Karrui
________________________________

**Comment by zenghou** [comment]
Thanks for the clarification @karrui! Prof @damithc, how are we supposed to update our team repo if we cannot merge the PPP PR because of integration issues?

**Comment by damithc** [comment]
> It's normal, converting it to a PDF allows everything to be properly displayed (But don't commit it to GitHub, Travis will complain)

@karrui what should not be committed and what's the Travis error?

@zenghou you can compare the following adoc file and corresponding html file:
* https://github.com/nus-cs2103-AY1718S1/addressbook-level4/blob/master/docs/team/johndoe.adoc
* https://nus-cs2103-ay1718s1.github.io/addressbook-level4/team/johndoe.html

**Comment by hymss** [comment]
It's the same for me too! The auto generated project portfolio html file will correctly display the excerpts of the user guide/ developer guide :-) but only commit the adoc file to github, not the auto generated html file.

**Comment by damithc** [comment]
> It's the same for me too! The auto generated project portfolio html file will correctly display the excerpts of the user guide/ developer guide :-) but only commit the adoc file to github, not the auto generated html file.

Yup, that's right. GithHub's adoc viewer doesn't know how to handle the `include` feature.  Yes, do not commit the pdf or the html files. From what I remember, html files (generated locally using gradel) go into a gitignored folder anyway.

**Comment by yunpengn** [comment]
The file will be rendered perfectly on your `gh-pages` branch, which has been deployed as GitHub Pages so it is in `.html` format.
